WebApp/Controllers/HomeController.cs

```diff
@@ -188,6 +188,21 @@ public async Task<ActionResult> ReadMail()
             return View();
         }
 
+        public async Task<ActionResult> ViewProfile()
+        {
+            IConfidentialClientApplication app = MsalAppBuilder.BuildConfidentialClientApplication();
+
+            var accountId = ClaimsPrincipal.Current.GetAccountId();
+            var account = await app.GetAccountAsync(accountId);
```

Comment:
I don't think you need to build a cca here, just show how to get and display (and delete) the SPA auth code.